13-securing-your-instance/README.md

```diff
@@ -128,7 +128,7 @@ Invenio works only with HTTPS so we create temporary certificates when starting
 
 ## Step 6: Configuration WSGI proxies
 
-The `docker-compose.full.yml` represents the common way Invenio is deployed, with two reverse proxies in front of the application. If you have a different number of proxies in front you should update your `WSGI_PROXY`, for more information read [here](https://invenio-base.readthedocs.io/en/latest/api.html#invenio_base.wsgi.wsgi_proxyfix).
+The `docker-compose.full.yml` demonstrates the common way Invenio is deployed, with two reverse proxies in front of the application. If you have a different number of proxies in front you should update your `WSGI_PROXY`, for more information read [here](https://invenio-base.readthedocs.io/en/latest/api.html#invenio_base.wsgi.wsgi_proxyfix).
```

Comment:
I would remove `the common way`.
```
The `docker-compose.full.yml` is an example of an Invenio application deployment. Since, in this case, each HTTP request passes through two servers, the HAProxy load balancer and the Nginx reserve proxy in front of the Invenio Flask application, we can set `WSGI_PROXY = 2`. If you have a different number of proxies in front of Invenio, you should update `WSGI_PROXY`. For more information read [here](https://invenio-base.readthedocs.io/en/latest/api.html#invenio_base.wsgi.wsgi_proxyfix).
```

Reply:
Fixed. I changed the "is deployed" by "could be" deployed. Since in the Invenio docs we put a note (see the other PR https://github.com/inveniosoftware/invenio/pull/3996) that says "this is just one possiblity, etc. etc."